src/main/resources/assets/pyrotech/lang/zh_cn.lang

```diff
@@ -327,13 +327,15 @@ gui.pyrotech.tooltip.uses.full=耐久：%d
 gui.pyrotech.tooltip.durability=耐久：%d / %d
 gui.pyrotech.tooltip.durability.full=耐久：%d
 gui.pyrotech.tooltip.durability.indestructible=Indestructible
+#I (DRAMING)couldn't find this entry, so I didn't translate it.(Could you please tell me?)
```

Comment:
The indestructible tooltip is used when a cog or sawblade is flagged as indestructible in the config file.

Reply:
Thanks!